src/app/public/modules/media-query/media-query-listener.ts

```diff
@@ -1,7 +1,7 @@
+// #region imports
 import {
   SkyMediaBreakpoints
 } from './media-breakpoints';
+// #endregion
 
-export interface SkyMediaQueryListener {
-  (args: SkyMediaBreakpoints): void;
-}
+export type SkyMediaQueryListener = (args: SkyMediaBreakpoints) => void;
```

Comment:
Why are we switching this to a `type`?

Reply:
<img width="587" alt="screen shot 2018-12-13 at 1 05 08 pm" src="https://user-images.githubusercontent.com/12497062/49958101-cddc7100-fed7-11e8-88b5-924a0198904f.png">

Reply:
I get that linting error with our more strict tslint.json file. Shouldn't affect the consumer.